_data/guides.yaml

```diff
@@ -99,6 +99,9 @@ categories:
       - title: Consuming and Implementing GraphQL Services
         url: /guides/microprofile-graphql
         description: This guide explains how to leverage Eclipse MicroProfile GraphQL to consume and implement GraphQL services.
+      - title: RESTEasy Reactive
+        url: /guides/rest-reactive
```

Comment:
I think it would be better for SEO if the URL was `resteasy-reactive`.

Reply:
Probably. I'm mostly putting this up to help the review of https://github.com/quarkusio/quarkus/pull/13802 so better you wait for @geoand and @stuartwdouglas 's first pass before reviewing this, the paint's still fresh :)